invenio_pages/serializers/response.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2020 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio module REST response serialisation."""
+
+import json
+
+from flask import current_app, request
+
+from invenio_pages import Page
+
+
+def _format_args():
```

Comment:
Our of curiosity, whey did you add this? browser and many CLI clients can prettify responses, no?

Reply:
for the cli mainly, because we had this somewhere as an option in the tutorials also. But you are right I can remove it